test/strategies-integration.test.ts

```diff
@@ -90,20 +91,18 @@ const testSuite = (sessionPluginName: string) => {
     })
 
     test('should initiate oauth with the openid-client strategy from npm', async () => {
-      const issuer = new OpenIdIssuer({ issuer: 'test_issuer', authorization_endpoint: 'http://www.example.com' })
-
-      const client = new issuer.Client({
+      const issuer = { issuer: 'https://as.example.com/', authorization_endpoint: 'https://as.example.com/authorize' }
+      const client = {
         client_id: 'identifier',
         client_secret: 'secure',
-        redirect_uris: ['http://www.example.com/auth/openid-client/callback']
-      })
+      }
 
-      const strategy = new OpenIdStrategy(
-        {
-          client
-        },
+      const config = new OpenIdClientConfiguration(issuer, client.client_id, client.client_secret)
+
+      const strategy = new OpenIdClientStrategy(
+        { config },
         () => fail()
-      ) as TestStrategy
+      ) as unknown as TestStrategy
```

Comment:
`TestStrategy` expects the following properties of any passed in strategy - success, fail, redirect, pass, and error, those are however properties added by passport, not individual strategies whos type is [much simpler](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3c4595ed1f8df849f2a46b0008b07715dbadd94f/types/passport/index.d.ts#L667-L676).